android/app/src/main/java/org/fossasia/openevent/common/utils/Utils.kt

```diff
@@ -0,0 +1,198 @@
+package org.fossasia.openevent.common.utils
+
+import android.content.Context
+import android.content.Intent
+import android.content.res.Resources
+import android.graphics.BitmapFactory
+import android.net.Uri
+import android.provider.CalendarContract
+import android.support.customtabs.CustomTabsIntent
+import android.support.v4.content.ContextCompat
+import android.support.v4.widget.SwipeRefreshLayout
+import android.util.Patterns
+import android.view.View
+
+import org.fossasia.openevent.R
+import org.fossasia.openevent.common.ConstantStrings
+import org.fossasia.openevent.common.api.Urls
+import org.fossasia.openevent.common.date.DateConverter
+import org.fossasia.openevent.config.StrategyRegistry
+import org.fossasia.openevent.data.Event
+
+import java.util.regex.Matcher
+import java.util.regex.Pattern
+
+object Utils {
+
+        @JvmStatic
+        var twoPane = false
+        set(value) {field = value}
+
+        @JvmStatic
```

Comment:
I don't think JvmStatic is required in objects

Reply:
We need that. Otherwise we get the error "Cannot be reference from a static context"